base/applications/mplay32/mplay32.c

```diff
@@ -1149,22 +1149,16 @@ OpenFileDialog(HWND hwnd, DWORD dwFilterIndex, LPTSTR lpType)
 {
     OPENFILENAME OpenFileName;
     TCHAR szFile[MAX_PATH + 1] = _T("");
-    TCHAR szCurrentDir[MAX_PATH];
 
     ZeroMemory(&OpenFileName, sizeof(OpenFileName));
 
-    if (!GetCurrentDirectory(ARRAYSIZE(szCurrentDir), szCurrentDir))
-    {
-        StringCbCopy(szCurrentDir, sizeof(szCurrentDir), _T("c:\\"));
-    }
-
     OpenFileName.lStructSize     = sizeof(OpenFileName);
     OpenFileName.hwndOwner       = hwnd;
     OpenFileName.hInstance       = hInstance;
     OpenFileName.lpstrFilter     = szFilter;
     OpenFileName.lpstrFile       = szFile;
     OpenFileName.nMaxFile        = ARRAYSIZE(szFile);
-    OpenFileName.lpstrInitialDir = szCurrentDir;
+    OpenFileName.lpstrInitialDir = NULL;
```

Comment:
You can then even remove that line completely since it is initialized to zeroes due to the ZeroMemory call from above.